control/excalibur/hl_detector.py

```diff
@@ -1246,12 +1263,19 @@ def status_loop(self):
                         self.update_calibration(self.STR_STATUS_LV_ENABLED, '1')
                     except:
                         pass
-            if (datetime.now() - self._slow_update_time).seconds > 10.0:
-                self._slow_update_time = datetime.now()
-                self.slow_read()
-            if (datetime.now() - self._medium_update_time).seconds > 10.0:
-                self._medium_update_time = datetime.now()
-                self.power_card_read()
+            if self._poll_active:
+                if (datetime.now() - self._slow_update_time).seconds > 10.0:
+                    self._slow_update_time = datetime.now()
+                    self.slow_read()
+                if (datetime.now() - self._medium_update_time).seconds > 10.0:
+                    self._medium_update_time = datetime.now()
+                    self.power_card_read()
+            else:
+                # Check for the poll disable timeout
+                if (datetime.now() - self._poll_timeout).total_seconds() > 60.0:
```

Comment:
Should this number 60.0 be a definition somewhere?  I know I probably wrote this so it's my own error, but it might be useful to have this value clearly defined somewhere (and maybe even the two 10.0 above also)

Reply:
There are a bunch of constants in this block of code and I am not sure what appropriate names are or which ones should be the same and which ones just happen to be the same. Also the comment at the top stating the poll rates does not match the code.

Reply:
OK I'll add this into the refactor I'm going to look at (mentioned below).  Lets get this PR merged so we are working from a consistent base.